cmd/qemu-ifdown/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+var DefaultConfPath string
+
+// todo: error handling
```

Comment:
I wasn't sure how much error handling was required here, it seems like the config itself has different standards that I didn't realize when I created this, so there may not be much error handling to do after all.